core/src/main/java/org/apache/calcite/sql/SqlFunctionCategory.java

```diff
@@ -65,8 +67,8 @@ public boolean isUserDefined() {
     return properties.contains(USER_DEFINED);
   }
 
-  public boolean isUserDefinedFunction() {
```

Comment:
Why are we deleting this existing function? Is it not used anymore?

Reply:
Yeah, I was initially using it in the previous feature but forgot to remove it.